docs/api-testing/on-prem/bloodhound.md

```diff
@@ -5,17 +5,19 @@ sidebar_label: Introduction
 description: "Bloodhound is an HTTP API Microgateway that acts are a reverse proxy."
 ---
 
+>**Legacy Documentation**: You're viewing the API Testing On-Premises (Legacy) documentation. To see the current documentation, see [API Testing on the Sauce Labs Cloud](/api-testing/).
+
```

Comment:
"You're viewing legacy documentation for API Fortress (deployed via an on-premises container). Find documentation for the new SaaS version of API Fortress - now known as Sauce Labs API Testing and Monitoring (with Sauce Connect tunnels) - at https://docs.saucelabs.com/api-testing/index.html.